samples/Facebook/README.md

```diff
@@ -0,0 +1,30 @@
+# Facebook Connector
+
+## Prerequisites
+
+1. Create a Facebook app `https://developers.facebook.com/docs/apps/#scenario`.
+2. Add 'Facebook Login' product to your app.
+3. Replace 'facebook_app_id' in apiProperties.json with your Facebook app id.
+
+    ```json
+      "clientId": "facebook_app_id"
+    ```
+
+4. Use CLI tool `paconn` to create a new custom connector.
+5. On Facebook developer dashboard update 'Valid OAuth Redirect URIs' under Products >> Facebook Login >> Settings.
+  You can find the redirect URI on security page of your custom connector.
+
+## Supported Actions
+
+As part of this sample following actions are supported:
+
+* `Get my Pages`: Lists all the pages that you own.
```

Comment:
Can you please add a bit info on what each operation does and how it works? e.g. we can add parameters does the operation take(if any), behind the scene what graph API gets called, how the response look like etc. Purpose is to educate someone trying to create connector for the first time